src/check/parse/IR.zig

```diff
@@ -48,6 +49,17 @@ pub const Diagnostic = struct {
         expected_platform_string,
         expected_package_or_platform_string,
         expected_package_platform_close_curly,
+        expect_closing_paren,
```

Comment:
Thank you! I planned to go through and add all of these (and more probably) when I finished the grammar